src/pickley/__init__.py

```diff
@@ -1038,3 +1069,26 @@ def represented(self):
 
         result.append("")
         return "\n".join(result)
+
+
+def latest_pypi_version(package_name, index):
+    package_name = PypiStd.std_package_name(package_name)
+    if package_name:
+        url = f"{index.rstrip('/')}/{package_name}/"
+        response = http_get(url)
+        if response:
+            versions = sorted(i.version for i in _parsed_simple_html(response) if i.version.is_final)
```

Comment:
are pre-only projects too much hassle?

Reply:
No pre-release only CLI ever came up so far. Handling pre-releases is possible, but I'd have to define "desired" version better... currently "latest non-pre-release" is the only thing "desired". I do plan to formalize this better in the future, with 3 kinds of "desired track":
- latest (default)
- pre-release (ie: latest not limited to final versions)
- pinned (everything else, tracked correctly, ie an `==` would be an absolute pin, an `~=` would be respected appropriately etc)